common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -5333,17 +5344,18 @@ is_not_isolated_from_prev_trigger = {
 			PREV = { is_in_pandaria_trigger = yes }
 		}
 	}
-}
-is_isolated_in_pandaria_trigger = {
-	NOT = { has_global_flag = mists_of_pandaria_parted_flag }
-	NOT = { has_game_rule = { name = mists_of_pandaria value = off } }
-	is_in_pandaria_trigger = yes
-}
-is_in_pandaria_or_no_mists = {
-	OR = {
-		has_global_flag = mists_of_pandaria_parted_flag
-		has_game_rule = { name = mists_of_pandaria value = off }
-		is_in_pandaria_trigger = yes
+	trigger_else = {
```

Comment:
That's a reason for the bug. It must be `trigger_else_if` with a limit similar to the Pandaria trigger above.